parquet-avro/src/test/java/org/apache/parquet/avro/TestAvroSchemaConverter.java

```diff
@@ -128,6 +216,9 @@ public void testAllTypes() throws Exception {
             "  optional group myoptionalarray (LIST) {\n" +
             "    repeated int32 array;\n" +
             "  }\n" +
+            "  required group myarrayofoptional (LIST) {\n" +
+            "    repeated int32 array;\n" +
```

Comment:
No, this is testing the old behavior and verifying that there is no change when the configuration options to turn on the new behavior are not set. Basically validating backward-compatibility for code.